aospy/test/data/objects/examples.py

```diff
@@ -1,23 +1,24 @@
 import os
```

Comment:
Sorry this was buried in the commit message of 36b526c.  They have been changed (only to add the ```time_bounds``` variable):

> Initial steps towards integrating DataLoader into Calc  …
This is far from finished, but the limited tests we have are passing. Note that I modified the example .nc files to include the CF-required time_bounds variable.